static/app/components/onboarding/frameworkSuggestionModal.tsx

```diff
@@ -76,33 +78,48 @@ export function FrameworkSuggestionModal({
   );
 
   useEffect(() => {
-    trackAnalytics('onboarding.select_framework_modal_rendered', {
-      platform: selectedPlatform.key,
-      organization,
-    });
-  }, [selectedPlatform.key, organization]);
+    if (newOrg) {
```

Comment:
Should we also track this for orgs that are not new?
Either under a different tracking key or the same one with different parameter.

Reply:
yes I plan to do that in a separate PR